src/EventSubscriber/CacheableResponseSubscriber.php

```diff
@@ -21,14 +22,42 @@ class CacheableResponseSubscriber implements EventSubscriberInterface {
    */
   protected $logger;
 
+  /**
+   * Configuration Factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactory
+   */
+  protected $configFactory;
+
   /**
    * Constructs a new DefaultExceptionHtmlSubscriber.
    *
    * @param \Psr\Log\LoggerInterface $logger
    *   The logger service.
+   * @param \Drupal\Core\Config\ConfigFactory $configFactory
+   *   Configuration for this module.
    */
-  public function __construct(LoggerInterface $logger) {
+  public function __construct(LoggerInterface $logger, ConfigFactory $configFactory) {
     $this->logger = $logger;
+    $this->configFactory = $configFactory;
```

Comment:
This change was made in compliance with Drupal coding standards. Local variables use underscores, not camel case.

```suggestion
    $this->configFactory = $config_factory;
```